org.mersenne.mprime.metainfo.xml

```diff
@@ -16,9 +17,13 @@
   <url type="homepage">https://www.mersenne.org</url>
   <developer_name>George Woltman</developer_name>
   <screenshots>
-          <screenshot>https://3.bp.blogspot.com/-CAWPIcQ9Qx8/T6vzdHj-B2I/AAAAAAAAAPY/6A_t46hl5mo/s1600/Screenshot+from+2012-05-10+19:35:57.png</screenshot>
+    <screenshot type="default">
+      <image type="source">https://3.bp.blogspot.com/-CAWPIcQ9Qx8/T6vzdHj-B2I/AAAAAAAAAPY/6A_t46hl5mo/s1600/Screenshot+from+2012-05-10+19:35:57.pn</image>
+      <caption>Main screen</caption>
+    </screenshot>
   </screenshots>
   <releases>
+    <release version="30.19b13" date="2024-03-07"/>
```

Comment:
Thank you!
Well, this seems to be the last remaining error…
But wait: this version is "30.19 (build 13)". Due to the recent versioning scheme you suggest to write "3019b13". But what about a "31.1 (build 10)" release, this would give "311b10". Or should it then be written as "3101b10"?
I am a bit confused and I am worried about correct versioning.
What about deleting all release info before this release in the metadata file?

Reply:
Yea you can delete too. But keep the one just before this and change it to 30.08